README.md

```diff
@@ -1,5 +1,6 @@
 # The yt Project
 
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/yt)
```

Comment:
note: we might want to hold off this line since it doesn't reflect the currently supported Python versions accurately (but I hope it will for the 4.0 release)